app/controllers/peeps.rb

```diff
@@ -0,0 +1,20 @@
+require 'sinatra/base'
+
+class Chitter < Sinatra::Base
+
+  get '/peeps' do
+    @peeps = Peep.all(:order => [:created_at.desc])
```

Comment:
It would be preferable to push this logic into the model for better encapsulation.  Then you could have something like `@peeps = Peep.in_reverse_order`